widget/ui/src/components/Chip/Chip.styles.ts

```diff
@@ -19,6 +19,10 @@ export const ChipContainer = styled('button', {
     backgroundColor: '$$color',
   },
 
+  '&:focus': {
```

Comment:
I think you can remove the outline in the selected state. in the current version selected chips have a border & outline simultaneously.